### DISCUSSION TITLE
Pages won't load properly after domain change.

### DISCUSSION THREAD

**Discussion by LarsIntraverte** [elementor/elementor#19945] [Developers Community]
Yesterday i changed the URL of my domain. Since then any changes i try to make to my website or new pages i add won't load the CSS properly. The weird thing is that in the Elementor editor everything looks normal. The images below show the Elementor editor (how it's supposed to look) and the page after publishing.

I have already updated the URL in the Elementor settings, regenerated the CSS files and emptied all my caches. I'm not experienced with code, so i don't know where i can find what the issue is. Hopefully someone can help me!

<img width="2048" alt="Schermafbeelding 2022-09-29 om 09 38 11" src="https://user-images.githubusercontent.com/114648950/192971065-3e6f4837-261d-4256-896b-3b940135ab37.png">
<img width="2048" alt="Schermafbeelding 2022-09-29 om 09 41 14" src="https://user-images.githubusercontent.com/114648950/192971070-41252e9a-17f4-4b8c-bf4a-f767ca11d50a.png">

I searched for a similar discussion on this topic, found [here](https://github.com/elementor/elementor/issues/12659), but changing one of the pages' image to the same image doesn't work for me.

Additional wordpress info:

<img width="1890" alt="Schermafbeelding 2022-09-29 om 09 49 15" src="https://user-images.githubusercontent.com/114648950/192972318-11a5a4c4-92eb-4879-abe5-da4d5317ca6d.png">
<img width="1890" alt="Schermafbeelding 2022-09-29 om 09 49 28" src="https://user-images.githubusercontent.com/114648950/192972309-c7f8e21c-4be2-48a0-854f-e5a7a34940a3.png">

**Comment by sinanisler** [comment]
this is The Color Bug and there are lots of similar bugs connected to this as well... 
@elementor refusing to fix this for a long long time :)

just set your colors again on the new site that's the fastest way to fix it.  ;)

**Comment by nicholaszein** [comment]
Try following this guide, most likely it will fix your issue:

https://www.facebook.com/groups/Elementors/permalink/2661661010626335/
